### ISSUE TITLE
Anti-La/SS-B antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6584]
**Preferred term label:** Anti-La/SS-B antibody positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Anti-extractable nuclear antibody positivity (when created)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Systemic Lupus Erythematosus OMIM:152700
Sjogren Syndrome OMIM:270150

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added
